src/data/cloud-computing-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const cloudComputingQuiz = [
+  {
+    Question: "Which of the following is NOT a common strategy in cloud migration?",
+    Answer: "Redecorating",
+    Distractor1: "Rehosting",
+    Distractor2: "Repurchasing",
+    Distractor3: "Refactoring",
+    Explanation: "Rehosting, repurchasing, and refactoring are common cloud migration strategies, each with its own approach to moving applications and services to the cloud. Redecorating is not a recognized strategy in cloud migration and is included as a distractor.",
```

Comment:
The explanation is unsatisfactory

Reply:
@Gautam-Hegde 

It would be more helpful to @Jasmine-maryj to explain what is missing from the explanation so they can make changes.
Otherwise, they won't know how to change it 👍

Reply:
oh I see your suggestion down below.

In general, I would then rephrase the feedback from unsatisfactory to "try this instead as it is more specific explanation to the question"
or something like that